src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -13,11 +13,17 @@ namespace System.Device.Gpio.Drivers
 {
     public unsafe partial class RaspberryPi3Driver : GpioDriver
     {
+
+        // error indicates that an entity doesn't exist
+        private const int Linux_Enoent = 2;
+
         private RegisterView* _registerViewPointer = null;
-        private const int GpioRegisterOffset = 0x00;
         private static readonly object s_initializationLock = new object();
         private static readonly object s_sysFsInitializationLock = new object();
+        private const uint GpioPeripheralOffset = 0x00200000; // offset from the peripheral base address of the GPIO registers
```

Comment:
NIT: can you use digit separators like `0x200_000`

Reply:
Addressed in latest PR